docs/docs/version-comparison.md

```diff
@@ -2,59 +2,126 @@
-- [Custom fields](/advanced-use/custom-fields.md) for features, segments and environments.
-- [Okta](/system-administration/authentication/Okta), [LDAP](/system-administration/authentication/LDAP),
-  [SAML](/system-administration/authentication/SAML) and [ADFS](/system-administration/authentication/ADFS)
-  authentication, as well as the ability to lock authentication to a single provider.
-- Additional database engines: Oracle and MySQL.
-- Additional deployment and orchestration options as detailed below.
+<div className="row">
+  <div className="col col--4">
+    <Card>
+      <CardHeader>
+        <h3>🚀 Free </h3>
+      </CardHeader>
+      <CardBody>
+        <ul>
+          <li>Up to 50,000 Requests/Month</li>
+          <li>1 Team Member</li>
+          <li>Unlimited Feature Flags</li>
+          <li>Unlimited Environments</li>
+          <li>A/B and MVT Testing</li>
+          <li>API Access</li>
+        </ul>
+      </CardBody>
+      <CardFooter>
+        <a href="/deployment" className="button button--primary button--sm">
+          View Deployment Guide
+        </a>
+      </CardFooter>
+    </Card>
+  </div>
+  
+  <div className="col col--4">
+    <Card>  
+      <CardHeader>
+        <h3>☁️ Start-Up</h3>
+      </CardHeader>
+      <CardBody>
+        <ul>
+          <li>Up to 1,000,000 Requests/Month</li>
+          <li>3 Team Members</li>
+          <li>Unlimited Projects</li>
+          <li>Email Technical Support</li>
+          <li>Scheduled Flags</li>
+          <li>Global Edge API</li>
+        </ul>
+      </CardBody>
+      <CardFooter>
+        <a href="https://app.flagsmith.com/signup" className="button button--primary button--sm">
+          Start Free Trial
+        </a>
+      </CardFooter>
+    </Card>
+  </div>
 
-## Open Source Benefits
+  <div className="col col--4">
+    <Card>
+      <CardHeader>
+        <h3>🏢 Enterprise</h3>
+      </CardHeader>
+      <CardBody>
+        <ul>
+          <li>5,000,000+ Requests/Month</li>
+          <li>20+ Team Members</li>
+          <li>Advanced Hosting Options</li>
+          <li>Priority Support via Slack/Discord</li>
+          <li>Custom SLA</li>
+        </ul>
+      </CardBody>
+      <CardFooter>
+        <a href="https://flagsmith.com/contact-us" className="button button--primary button--sm">
+          Contact Sales
+        </a>
+      </CardFooter>
+    </Card>
+  </div>
+</div>
 
-- Completely Free!
-- The Open Source version has **no** API request or Identity limits - you can run as many API instances in a cluster as
-  you wish.
-- The Open Source version has **no** Dashboard User limits - you can have as many team members as you wish.
-- Deploy with one click to a number of different [IaaS and PaaS providers](/deployment#one-click-installers).
+:::tip Migration
+You can easily switch between SaaS and Self Hosted Flagsmith using our [Import and Export tools](system-administration/importing-and-exporting/organisations).
```

Comment:
I would leave out "easily" here. There is a decent amount of manual work involved in a migration, and we don't want to encourage customers to depend on this. Migration is more like an escape hatch that's available if a customer ever decides they need to switch.